recipes/avro/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.7.7" %}
+
+package:
+  name: avro
+  version: {{ version }}
+
+source:
+  fn: avro-1.7.7.tar.gz  # [py27]
+  url: https://pypi.python.org/packages/source/a/avro/avro-1.7.7.tar.gz  # [py27]
+  md5: e21ae1dffea4c96ff151ce35ca5a8875  # [py27]
+
+  fn: avro-python3-1.7.7.tar.gz  # [py3k]
+  url: https://pypi.python.org/packages/source/a/avro-python3/avro-python3-1.7.7.tar.gz  # [py3k]
+  md5: bd36c1e6fbb77ccff58c60bed48b982e  # [py3k]
```

Comment:
The python avro library does not use something like six to be cross platform.  They instead made a copy of it using 2-3 and fixed pieces manually.